package.json

```diff
@@ -5,15 +5,15 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "watcher:signature-request": "node src/watcher.js signature-request-watcher.config.js",
-    "watcher:collected-signatures": "node src/watcher.js collected-signatures-watcher.config.js",
-    "watcher:affirmation-request": "node src/watcher.js affirmation-request-watcher.config.js",
-    "sender:home": "node src/sender.js home-sender.config.js",
-    "sender:foreign": "node src/sender.js foreign-sender.config.js",
+    "watcher:signature-request": "node src/watcher.js signature-request-watcher.config.js | tee -a logs/watcher-signature-request.txt | pino",
+    "watcher:collected-signatures": "node src/watcher.js collected-signatures-watcher.config.js | tee -a logs/watcher-collected-signatures.txt | pino",
+    "watcher:affirmation-request": "node src/watcher.js affirmation-request-watcher.config.js | tee -a logs/watcher-affirmation-request.txt | pino",
+    "sender:home": "node src/sender.js home-sender.config.js | tee -a logs/sender-home.txt | pino",
+    "sender:foreign": "node src/sender.js foreign-sender.config.js | tee -a logs/sender-foreign.txt | pino",
     "dev": "concurrently -n 'watcher:signature-request,watcher:collected-signatures,watcher:affirmation-request,sender:home,sender:foreign' -c 'red,green,yellow,blue,magenta' 'npm run watcher:signature-request' 'npm run watcher:collected-signatures' 'npm run watcher:affirmation-request' 'npm run sender:home' 'npm run sender:foreign'",
     "test": "NODE_ENV=test mocha",
     "coverage": "NODE_ENV=test nyc --reporter=text --reporter=html mocha",
-    "postinstall": "npm install --prefix e2e"
+    "postinstall": "npm install --prefix e2e && mkdirp logs"
```

Comment:
Could you clarify why `mkdirp` is used instead of generic `mkdir` command?

Reply:
No reason, really. `mkdirp` doesn't fail if the directory exists, but that can also be accomplished with `mkdir -p` (hence the module name). It's also more "portable", but I also used tee, so that's not a valid reason either. I will just use `mkdir` so that there are less dependencies.

Reply:
Please check the last changes to see if you are OK with that approach (I added a bash script that just checks the NODE_ENV variable and prettifies the output if the environment is not production).